extensions/amp-story/1.0/test/test-amp-story-request-service.js

```diff
@@ -79,7 +88,7 @@ describes.fakeWin('amp-story-request-service', {amp: true}, (env) => {
       })
       .once();
 
-    await requestService.loadShareConfig();
+    await requestService.loadConfigImpl(shareElement);
```

Comment:
This test should fail now, as you removed the code that only fetches once when called multiple times